go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/ably/ably-go
 
+go 1.18
+
```

Comment:
Did you intend to include this change? It seems unrelated

Reply:
The new tests won't compile without this because it includes the `any` type, which was introduced in 1.18.
If you think it is important, I could change all occurrences of `any` to `[]byte`

Reply:
Ok thanks, so I think this should be in a separate commit with that context in the commit message.

Reply:
I will squash merge this anyway, so the commits will be mushed together into one commit.
Should I do a separate PR for this?

Reply:
No separate PR no, just merge with two commits?